src/config/proxy.ts

```diff
@@ -1,15 +1,27 @@
 import { envVars }  from "./env";
-import { bootstrap } from "global-agent";
-import { getLogger } from "./logger";
-
-const logger = getLogger("config.proxy");
-
-if (envVars.PROXY) {
-	logger.info(`configuring proxy: ${envVars.PROXY} for outbound calls`);
-	process.env.GLOBAL_AGENT_HTTP_PROXY = envVars.PROXY;
-	bootstrap();
-} else {
-	logger.info("configuring no proxy for outbound calls");
-}
+import { AxiosRequestConfig } from "axios";
+import { HttpsProxyAgent } from "https-proxy-agent";
+
+const outboundProxyHttpsAgent = envVars.PROXY ? new HttpsProxyAgent(envVars.PROXY) : undefined;
+
+/**
+ * Use this Axios config to make HTTP(S) requests via the configured outbound proxy.
+ */
+export const OUTBOUND_PROXY_CONFIG: Partial<AxiosRequestConfig> = {
+	// Even though Axios provides the `proxy` option to configure a proxy, this doesn't work and will
+	// always cause an HTTP 501 (see https://github.com/axios/axios/issues/3459). The workaround is to
```

Comment:
that thread at the end said to update to the latest version as it should be fixed now?

Reply:
I'll try that.

Reply:
The comment "Please could you retry with the latest version and open a new issue should this error still be relevant?" combined with the status "possible bug" don't spark confidence in me, though :)

Reply:
Nope, not working:
<img width="866" alt="image" src="https://user-images.githubusercontent.com/1363578/179918632-b448a456-b01c-47e4-9b53-4b5b6a3b8242.png">